src/components/Book.js

```diff
@@ -1,35 +1,48 @@
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
 const Book = (props) => {
   const { bookInfo, index } = props;
+  const {
+    title, author, category, progress, currentChapter,
+  } = bookInfo;
+  const dispatch = useDispatch();
+
+  const deleteBook = (e) => {
+    e.preventDefault();
+    dispatch(removeBook(title));
+  };
```

Comment:
- [ ] While removing a book, it is usually better to pass the `id` from the book data instead of the `title` while dispatching the remove process. Remember, some books may share the same title, but while using the `uuidv4` for unique id for each book, all books will the have different id's.